### PR TITLE
Initial update to Readme in new repo

### PR DESCRIPTION
**Pull request by bethz**
Many word updates:
- step by step
- badges for pipeline tests
- badges for azure notebooks and binder (may be removed later if widget does not work)

Even though this is a WIP, it is being put in as a PR for visibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,154 @@
...
+# Models
+
+# Mehrnoosh todo: this section requires rewording
+Add ref to
+https://docs.microsoft.com/en-us/python/api/azureml-explain-model/azureml.explain.model?view=azure-ml-py
+
+* The API supports both dense (numpy or pandas) and sparse (scipy) datasets
+
+* For more advanced users, individual explainers can be used
+ (details?)
```
Comment (imatiach-msft) [inline README.md]: remove (details?)

File: README.md [hunk 2]
```diff
@@ -1,26 +1,154 @@
...
+        - Best implies fastest execution time and highest interpretabilty accuracy.
+* Local feature importances are for each evaluation row
+* Global feature importances summarize the most importance features at the model-level
  * The KernelExplainer and MimicExplainer are for BlackBox models
  * The MimicExplainer is faster but less accurate than the KernelExplainer
  * The TreeExplainer is for tree-based models
  * The DeepExplainer is for DNN tensorflow or pytorch models
+[shap](https://github.com/slundberg/shap) and [lime](https://github.com/marcotcr/lime) have docs
+
+global surrogate is ???
```
Comment (imatiach-msft) [inline README.md]: let's not mention this for now

File: README.md (hunk 1 above)
Comment (bethz) [reply README.md]: removed.

File: README.md (hunk 2 above)
Comment (bethz) [reply README.md]: ok.

**Pull request merged**